lib/src/components/NxList/types.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {
+  HTMLAttributes,
+  LiHTMLAttributes,
+  ValidationMap,
+  ReactNode,
+  ButtonHTMLAttributes,
+  AnchorHTMLAttributes
+} from 'react';
+import PropTypes from 'prop-types';
+
+export interface NxListProps extends HTMLAttributes<HTMLUListElement> {
+  bulleted?: boolean | null;
+  emptyMessage?: ReactNode;
+  error?: string | null;
+  isLoading?: boolean | null;
+  retryHandler?: (() => void) | null;
+}
+
+export const nxListPropTypes: ValidationMap<NxListProps> = {
+  bulleted: PropTypes.bool,
+  emptyMessage: PropTypes.node,
+  error: PropTypes.string,
+  isLoading: PropTypes.bool,
+  retryHandler: PropTypes.func
+};
+
+export interface NxListButtonItemProps extends LiHTMLAttributes<HTMLLIElement> {
+  selected?: boolean | null,
+  disabled?: boolean | null,
+  buttonClassName?: string | null,
+  buttonAttributes?: ButtonHTMLAttributes<HTMLButtonElement> | null,
+}
+
+export const nxListButtonItemPropTypes: ValidationMap<NxListButtonItemProps> = {
```

Comment:
This and `nxListLinkItemPropTypes` are missing some fields.  I'm surprised this passed the linter.

Reply:
> This and `nxListLinkItemPropTypes` are missing some fields. I'm surprised this passed the linter.

The missing fields have been added.
850dac63599cf8a0524ebae3d10eb023268c6fca